docs/docsite/rst/network/dev_guide/developing_plugins_network.rst

```diff
@@ -0,0 +1,108 @@
+
+.. _developing_modules_network:
+.. _developing_plugins_network:
+
+**************************
+Network connection plugins
+**************************
+
+Each network connection plugin has a set of plugins which provide a specification of the connection for a particular set of devices. Public methods of these plugins may be called on the connection proxy object from the module just as connection methods can.
+
+.. code-block:: python
+
+  from ansible.module_utils.connection import Connection
+
+  ...
+
+  # module is your AnsibleModule instance
+  connection = Connection(module._socket_path)
```

Comment:
I am receiving errors around `module._socket_path` not being found (don't have the error in front of me). I'd like to see more documentation about configuring `_socket_path`. I've also noticed not all plugin examples appear to use it (FTD?) so more detail about that may be helpful too.